inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -480,9 +473,14 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     if (!ngraphNode && opsets.count(params.version)) {
         auto opset = opsets.at(params.version);
         std::string type = params.type;
+
         if (type == "Const") {
             type = "Constant";
         }
+        // MVN was missing in opset1
+        if (type == "MVN" && params.version == "opset1") {
```

Comment:
Can we use one`if` for `MVN` and `ROIPooling`?

Reply:
Sure!